src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -46,8 +48,8 @@ function informationDiagram(publicAPI, model) {
 
   model.clientRect = null;
 
-  // FIXME: Make some attempt at unique id, for now just use millis timestamp
-  model.instanceID = `informationDiagram-${Date.now()}`;
+  // unique id, based on millis timestamp
+  model.instanceID = `informationDiagram-${generateUUID()}`;
```

Comment:
Why not just a global counter within the module? No complexity and no risk of same id.